### DISCUSSION TITLE
Property 'chainNamespace' is missing in type 'WagmiAdapter' but required in type 'ChainAdapter'.ts(2741)

### DISCUSSION THREAD

**Discussion by certifirm** [WalletConnect/walletconnect-monorepo#5559] [Developer Support]
const wagmiAdapter = new WagmiAdapter({
      projectId: PROJECT_ID,
      networks,
    });

    // 3. Create the wallet
    this.modal = createAppKit({
      allWallets: 'HIDE',
      adapters: [wagmiAdapter],
      networks: [bsc, optimism, base, arbitrum, polygon],
      metadata,
      projectId: PROJECT_ID,
      features: { ...........
      
I receive the error:
Property 'chainNamespace' is missing in type 'WagmiAdapter' but required in type 'ChainAdapter'.ts(2741), but I can not set chainNameSpace.  I am using angular.  With @reown/appkit-adapter-wagmi@1.3 version 1.3 works well.  but 1.4, 1.5, 1.6 not.

How can I set a chainNameSpace?

**Comment by rtomas** [comment]
hi @certifirm, if you are trying to define a custom network. Please check our docs

https://docs.reown.com/appkit/react/core/custom-networks
